avalonmediasystem/react-structural-metadata-editor
# Upgrade Peaks.js to latest stable version: 0.26.0

This needs to be tested on a Google Chrome browser, for me it works on both Safari and Firefox.

To test it out;
1. Pull the changes in the branch into local machine
2. `yarn install`
3. `yarn start`
4. Connect to IU VPN
5. Go to http://localhost:3001
6. Check whether the loading spinner stops and the streaming works
7. Check editing timespans and headings work
